6.py

```diff
@@ -0,0 +1,35 @@
+# 15596
+def solve(a):
+    return sum(a)
+
+# 4673
+# set의 특성은 '중복 숫자가 없다' -> 중복 제거하기 위한 필터 역할로 종종 사용 가능
+# set 이용하지 않을 때(리스트형으로 받아 주고 마지막 for문을 통해 중복 여부 확인)
+def d(n):
+    a = 0
+    for x in list(str(n)):
```

Comment:
문자열(str)형 자료형의 경우 이미 iterable한 객체이기 때문에 list로 형변환할 필요는 없어요
```suggestion
    for x in str(n):
```